src/Components/Slideshow/Slideshow.js

```diff
@@ -10,185 +10,140 @@ import './Slideshow.css'
 
 const items = [
   {
-    id: 0,
     src: 'images/officers2019.jpg',
     altText: 'Officers',
     header: '2019-2020 Officers',
-    caption: '',
     top: '25%'
   },
   {
-    id: 1,
     src: 'images/funnyofficers.jpg',
     altText: 'Officers',
     header: '2019-2020 Officers',
-    caption: '',
     top: '25%'
   },
   {
-    id: 2,
     src: 'images/Tesla2.jpg',
     altText: 'Tesla Visit',
     header: 'Tesla Tour',
-    caption: '',
     top: '60%'
   },
   {
-    id: 3,
     src: 'images/meeting2.jpg',
     altText: 'SCE General Meeting',
     header: 'General Meeting',
-    caption: '',
     top: '40%'
   },
   {
-    id: 4,
     src: 'images/sap.jpg',
     altText: 'SAP Tour',
     header: 'Company Tour',
-    caption: '',
     top: '65%'
   },
   {
-    id: 5,
     src: 'images/react.jpg',
     altText: 'React Workshop',
     header: 'React Workshop',
-    caption: '',
     top: '40%'
   },
   {
-    id: 6,
     src: 'images/chess.jpg',
     altText: 'Game Night!',
     header: 'Game Night',
-    caption: '',
     top: '60%'
   },
   {
-    id: 7,
     src: 'images/making-cake.jpg',
     altText: 'Cake Team Building',
     header: 'Cake Team Building',
-    caption: '',
     top: '40%'
   }
   /* Can add more images below as needed, or use Facebook API in the future to auto load images */
   /* Images must be 16:9 or 4:3 with minimum width of 1000 px */
 ]
 
-class Slideshow extends Component {
-  constructor (props) {
-    super(props)
-    this.state = { activeIndex: 0 }
-    this.handleNext = this.handleNext.bind(this)
-    this.handlePrevious = this.handlePrevious.bind(this)
-    this.onNext = this.handleNext.bind(this)
-    this.onPrevious = this.handlePrevious.bind(this)
-    this.handleGoToIndex = this.handleGoToIndex.bind(this)
-    this.handleOnExiting = this.handleOnExiting.bind(this)
-    this.handleOnExited = this.handleOnExited.bind(this)
+function Slideshow () {
+  const [currentSlide, setCurrentSlide] = useState(0)
+  const [animating, setAnimating] = useState(false)
+  function handleOnExiting () {
+    setAnimating(true)
   }
 
-  handleOnExiting () {
-    this.animating = true
+  function handleOnExited () {
+    setAnimating(false)
   }
 
-  handleOnExited () {
-    this.animating = false
+  function onNext () {
+    handleNext()
   }
 
-  onNext () {
-    this.handleNext()
+  function onPrevious () {
```

Comment:
this function should be removed, all it does is forward the call to `handlePrevious()`